src/client/ui/selector-inspector-panel/hide-button.js

```diff
@@ -0,0 +1,38 @@
+import hammerhead from './../deps/hammerhead';
+import testCafeCore from './../deps/testcafe-core';
+import { createElementFromDescriptor } from './utils/create-element-from-descriptor';
+import { setStyles } from './utils/set-styles';
+
+import * as descriptors from './descriptors';
+
+const nativeMethods = hammerhead.nativeMethods;
+
+const eventUtils = testCafeCore.eventUtils;
+
+const VALUES = {
+    show: 'Hide',
+    hide: 'Show',
+};
+
+export class HideButton {
+
+    constructor (element) {
+        this.element       = createElementFromDescriptor(descriptors.hideButton);
+        this.hideElement   = element;
+        this.isHide        = true;
+
+        eventUtils.bind(this.element, 'click', () => this._showAndHide());
+    }
+
+    _showAndHide () {
+        this._changeAppearance(this.isHide ? VALUES.hide : VALUES.show, 'bold');
+        setStyles(this.hideElement, this.isHide ? { display: 'none' } : { display: 'flex' });
+        this.isHide = !this.isHide;
+    }
+
+    _changeAppearance (value, fontWeight) {
+        nativeMethods.inputValueSetter.call(this.element, value);
+
+        setStyles(this.element, { fontWeight });
```

Comment:
We can just add additional class to the parent of the `hide` button.
Then, by clicking on the `hide` button. We can just toggle this class and manipulate by its children styles: hide button and main container.
This will allow us to remove `changeAppearance` and `setStyles` functions and `this.isHide` property and move all style related things into css
Also, at this moment, I think we do not need to add some value (Hide, Show) change logics, because it's likely that we add some icon to this in future.